RockPaperScissors/Program.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace RockPaperScissors
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Player player1 = new Player(0);
+            Player player2 = new Player(0);
+
+            List<string> whoWon = new List<string>();
```

Comment:
might be better to hard code an array `var whoWon = new string[]{"player 1", etc...}